ledger-tool/src/main.rs

```diff
@@ -1169,21 +1184,31 @@ fn main() {
         .multiple(true)
         .takes_value(true)
         .help("Log when transactions are processed that reference the given key(s).");
-    let boot_from_local_state = Arg::with_name("boot_from_local_state")
-        .long("boot-from-local-state")
-        .takes_value(false)
+    let default_use_snapshot_archives_at_startup = UseSnapshotArchivesAtStartup::Always.to_string();
+    let use_snapshot_archives_at_startup = Arg::with_name("use_snapshot_archives_at_startup")
+        .long("use-snapshot-archives-at-startup")
+        .takes_value(true)
+        .possible_values(&UseSnapshotArchivesAtStartup::variants())
+        .default_value(&default_use_snapshot_archives_at_startup)
+        .case_insensitive(true)
```

Comment:
?

Reply:
pretty sure we don't specify it elsewhere 'cause we don't use `arg_enum` anywhere else (besides dos tool) and use Industry Standard :tm: kebab-case throughout